pkgs/os-specific/linux/zfs/unstable.nix

```diff
@@ -23,9 +23,9 @@ callPackage ./generic.nix args {
   # IMPORTANT: Always use a tagged release candidate or commits from the
   # zfs-<version>-staging branch, because this is tested by the OpenZFS
   # maintainers.
-  version = "2.2.0-rc5";
+  version = "2.2.0";
```

Comment:
https://github.com/openzfs/zfs/tree/zfs-2.2.1-staging

A staging branch now exists for the next release

Reply:
@adamcstephens can you upgrade to the staging branch?

Reply:
I can, but it's the same exact commit as 2.2.0, so while it would imply it is a different version it wouldn't be yet.

https://github.com/openzfs/zfs/commits/zfs-2.2.1-staging/

I will still update this if desired, but until the branch is updated it will just be 2.2.0.

Reply:
I would prefer the change indeed.

Reply:
Seems disingenuous and confusing to me, but this is done.

Reply:
I think it makes it clear with the comment we are staying on staging again after the major release IMHO.